tests/issues/issue-468/issue-468.test.ts

```diff
@@ -0,0 +1,15 @@
+import { compareFiles } from 'tests/common';
+import { describe, expect, it } from 'vitest';
+
+describe('Issues', () => {
+  it('should end with newline in svelte', async () => {
+    const { actual, expected } = await compareFiles(import.meta.url, {
+      source: 'unformatted.svelte',
+      target: 'formatted.svelte',
+      formatOptions: {
+        pugFramework: 'svelte',
```

Comment:
When I add `parser: 'svelte'` here I get an error 🤔 
If I leave it, it seems to work

Reply:
ping @rodshtein

Reply:
Yeah, the previous version of plugin for svelte `prettier-plugin-svelte3` worked great.
I need a little time to dig deeper, I'll text you later.